### PR TITLE
Fixing basic MIME types

### PR DESCRIPTION
**Pull request by Ellerbach**
Fixing basic MIME types

## Motivation and Context

- Fixes nanoFramework/Home#1214

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [x] Bug fix (non-breaking change which fixes an issue with code or algorithm)
- [ ] New feature (non-breaking change which adds functionality to code)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (add new Unit Test(s) or improved existing one(s), has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project (only if there are changes in source code).
- [ ] My changes require an update to the documentation (there are changes that require the docs website to be updated).
- [ ] I have updated the documentation accordingly (the changes require an update on the docs in this repo).
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [x] I have tested everything locally and all new and existing tests passed (only if there are changes in source code).
- [ ] I have added new tests to cover my changes.

### CODE REVIEW

**Comment by DaveSchmid** [comment]
@Ellerbach you better rename filename to extension in GetContentTypeFromFileName(string fileName)

**Comment by josesimoes** [comment]
Guess this supersedes #213 correct?

File: nanoFramework.WebServer/nanoFramework.WebServer.nfproj [hunk 1]
```diff
@@ -91,6 +91,9 @@
   <ItemGroup>
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup>
```
Comment (josesimoes) [inline nanoFramework.WebServer/nanoFramework.WebServer.nfproj:94]: If you're adding this here, need to include the packages.lock.json file to the PR.

File: nanoFramework.WebServer/WebServer.cs [hunk 2]
```diff
@@ -468,7 +468,8 @@ public static void SendFileOverHTTP(HttpListenerResponse response, StorageFile s
         /// /// <param name="contentType">The type of file, if empty string, then will use auto detection</param>
```
Comment (josesimoes) [inline nanoFramework.WebServer/WebServer.cs:468]: "double comment" here
```suggestion
        /// <param name="contentType">The type of file, if empty string, then will use auto detection</param>
```

File: nanoFramework.WebServer/WebServer.cs [hunk 3]
```diff
@@ -371,7 +371,7 @@ public bool Start()
         /// <summary>
         /// Restart the server.
         /// </summary>
-        private bool Restart()
+        public bool Restart()
```
Comment (josesimoes) [inline nanoFramework.WebServer/WebServer.cs:374]: This one and the all the others in the comments are unrelated with the PR: fixing the MIME types. Can you please remove them and submit a separate PR?

**Review by josesimoes — COMMENTED** [review]
LGTM! Just please remove the changes unrelated with the PR.

File: nanoFramework.WebServer/nanoFramework.WebServer.nfproj (hunk 1 above)
Comment (Ellerbach) [reply nanoFramework.WebServer/nanoFramework.WebServer.nfproj:94]: it is already present as files

File: nanoFramework.WebServer/WebServer.cs (hunk 3 above)
Comment (Ellerbach) [reply nanoFramework.WebServer/WebServer.cs:374]: ok, made this one private again, for the rest of the comments, that was breaking my build somehow (warnings are errors)

Comment (josesimoes) [reply nanoFramework.WebServer/WebServer.cs]: OK. So let's have those fixed first and merge the PR into this one.

**Comment by josesimoes** [comment]
Call me picky, but I would rather have had the fixes for the comments in a separate PR...
